### PR TITLE
[Doc] Add docs for resource usage stats

### PR DESCRIPTION
**Pull request by bharath-techie**
### Description
Update docs for resource usage stats introduced as part of https://github.com/opensearch-project/OpenSearch/pull/9890

### Issues Resolved

Fixes https://github.com/opensearch-project/documentation-website/issues/5409 - Doc issue

Related to https://github.com/opensearch-project/OpenSearch/issues/8910 - Opensearch meta issue

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/nodes-apis/nodes-stats.md [hunk 1]
```diff
@@ -1143,6 +1153,15 @@ total_rejections_breakup_shadow_mode.throughput_degradation_limits | Integer | T
 enabled | Boolean | Specifies whether the shard indexing pressure feature is turned on for the node.
 enforced | Boolean | If true, the shard indexing pressure runs in enforced mode (there are rejections). If false, the shard indexing pressure runs in shadow mode (there are no rejections, but statistics are recorded and can be retrieved in the `total_rejections_breakup_shadow_mode` object). Only applicable if shard indexing pressure is enabled. 
 
+### `resource_usage_stats`
+The `resource_usage_stats` object contains the resource usage statistics. Each entry is specified by the node ID and has the following properties.
+
+Field | Field type | Description
+:--- |:-----------| :---
+timestamp | Integer    | The last refresh time for the resource usage statistics, in milliseconds since the epoch.
+cpu_utilization_percent | Float      | Statistics for the average CPU usage of OpenSearch process in the period configured in the setting `node.resource.tracker.global_cpu_usage.window_duration`.
```
Comment (ajaymovva) [inline _api-reference/nodes-apis/nodes-stats.md:1162]: Can we move field type to `double` instead of `float`.

**Comment by bharath-techie** [comment]
Hi @Naarcha-AWS @kolchfa-aws ,
Can you please review this PR ?

**Review by ajaymovva — APPROVED** [review]
LGTM.

File: _api-reference/nodes-apis/nodes-stats.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ adaptive_selection | Statistics about adaptive replica selection, which selects
 script_cache | Statistics about script cache.
 indexing_pressure | Statistics about the node's indexing pressure.
 shard_indexing_pressure | Statistics about shard indexing pressure.
+resource_usage_stats | Resource usage statistics of nodes, such as CPU and JVM
```
Comment (kolchfa-aws) [inline _api-reference/nodes-apis/nodes-stats.md:54]:
```suggestion
resource_usage_stats | Node-level resource usage statistics, such as CPU and JVM memory.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 3]
```diff
@@ -729,6 +737,8 @@ http.total_opened | Integer | The total number of HTTP connections the node has
 [indexing_pressure](#indexing_pressure) | Object | Statistics related to the node's indexing pressure.
 [shard_indexing_pressure](#shard_indexing_pressure) | Object | Statistics related to indexing pressure at the shard level.
 [search_backpressure]({{site.url}}{{site.baseurl}}/opensearch/search-backpressure#search-backpressure-stats-api) | Object | Statistics related to search backpressure.
+[resource_usage_stats](#resource_usage_stats) | Object | Statistics related to resource usage stats for the node.
```
Comment (kolchfa-aws) [inline _api-reference/nodes-apis/nodes-stats.md:740]:
```suggestion
[resource_usage_stats](#resource_usage_stats) | Object | Statistics related to resource usage for the node.
```

File: _api-reference/nodes-apis/nodes-stats.md (hunk 1 above)
Comment (kolchfa-aws) [inline _api-reference/nodes-apis/nodes-stats.md:1162]:
```suggestion
cpu_utilization_percent | Float      | Statistics for the average CPU usage of OpenSearch process within the time period configured in the `node.resource.tracker.global_cpu_usage.window_duration` setting.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 4]
```diff
@@ -1143,6 +1153,15 @@ total_rejections_breakup_shadow_mode.throughput_degradation_limits | Integer | T
...
 enforced | Boolean | If true, the shard indexing pressure runs in enforced mode (there are rejections). If false, the shard indexing pressure runs in shadow mode (there are no rejections, but statistics are recorded and can be retrieved in the `total_rejections_breakup_shadow_mode` object). Only applicable if shard indexing pressure is enabled. 
 
+### `resource_usage_stats`
+The `resource_usage_stats` object contains the resource usage statistics. Each entry is specified by the node ID and has the following properties.
+
+Field | Field type | Description
+:--- |:-----------| :---
+timestamp | Integer    | The last refresh time for the resource usage statistics, in milliseconds since the epoch.
+cpu_utilization_percent | Float      | Statistics for the average CPU usage of OpenSearch process in the period configured in the setting `node.resource.tracker.global_cpu_usage.window_duration`.
+memory_utilization_percent | Float      | Statistics for the JVM memory usage on the node in the period configured in the setting `node.resource.tracker.global_jvmmp.window_duration`
```
Comment (kolchfa-aws) [inline _api-reference/nodes-apis/nodes-stats.md:1163]:
```suggestion
memory_utilization_percent | Float      | The node JVM memory usage statistics within the time period configured in the `node.resource.tracker.global_jvmmp.window_duration` setting.
```

**Review by kolchfa-aws — APPROVED** [review]
Thanks, @bharath-techie! Left some rewrite comments.

**Review by vagimeli — APPROVED** [review]
No edits. Ready for publishing.

**Comment by bharath-techie** [comment]
Hi @kolchfa-aws @vagimeli 

When will we merge this PR ?

**Pull request merged**